docs/install/managing-multiple-projects.md

```diff
@@ -1,3 +1,103 @@
 # Managing Multiple Projects
 
-Documentation coming soon! See https://github.com/google/knative-gcp/issues/460
+A general Knative-GCP installation configures three identities (represents as Google service accounts) on your project.
+
+1. Identity for the Control Plane.
+2. Identity for the GCP Broker.
+3. Identity for the Source. (This is the default identity for the Source.
+Users can use this identity for multiple Source custom objects,
+or create additional identities in the form of Google service accounts for different Source custom objects,
+to achieve more fine-grained configurations)
+
+In general Knative-GCP installation guide, each Knative-GCP component consumes/creates Google Cloud resources within the scope of the project
+(the project where the kubernetes cluster contained your Knative-GCP components resides). For example, if your kubernetes cluster is in Project A, and
+you apply StorageSource in this kubernetes cluster, it will create a Google Cloud Storage Bucket, a Google Cloud PubSub Topic, and a Google Cloud PubSub Subscription,
+all belong to Project A.
+
+This topic explains how to set-up Knative GCP configuration to create Sources which consume/create Google Cloud resources from other projects.
+
+Here we use PubSubSource as an example.
+
+The following names are representing:
```

Comment:
```suggestion
We will use the following values as examples, you should replace them with the appropriate values for your project when running the example commands:
```

It might be worthwhile doing these as shell variables instead, something like:

```
# Cluster project is the GCP project that the Kubernetes cluster is running in.
export CLUSTER_PROJECT=project-a

# Source project is the GCP project that the source is reading information from.
export SOURCE_PROJECT=project-b

# Control plane GSA is the GSA that is being used by the control plane. Note that it is normally in the CLUSTER_PROJECT.
export CONTROL_PLANE_GSA="events-controller-gsa@$CLUSTER_PROJECT.iam.gserviceaccount.com"

# Source data plane GSA is the GSA that is used when reading from the source's PullSubscription. Note that it is normally in the SOURCE_PROJECT.
export SOURCE_DATA_PLANE_GSA="your-gsa@$SOURCE_PROJECT.iam.gserviceaccount.com"
```

Then the command below use the shell variables, rather than the example values.